app/views/application/_navbar.html.erb

```diff
@@ -1,19 +1,19 @@
 <%# Bootstrap navbar helper provided by twitter-bootstrap-rails gem  %>
 <%# https://github.com/seyhunak/twitter-bootstrap-rails#navbar-helper %>
-<%= nav_bar brand: t('appname'), responsive: true do %>
+<%= nav_bar brand: t('appname'), responsive: true, fluid: true do %>
   <%= menu_group do %>
     <%= menu_item User.model_name.human(:count => :many).titleize, users_path %>
     <%= menu_item Team.model_name.human(:count => :many).titleize, teams_path %>
     <%= menu_item Match.model_name.human(:count => :many).titleize, matches_path %>
     <%= menu_item Event.model_name.human(:count => :many).titleize, events_path %>
-    <%= menu_divider %>
+  <% end %>
+  <%= menu_group pull: :right do %>
     <% if user_signed_in? %>
       <%= menu_item current_user.first_name, user_path(current_user) %>
       <%= menu_divider %>
       <%= menu_item t('devise.registrations.sign_out'), destroy_user_session_path, method: :delete %>
     <% else %>
-      <%= menu_item t('devise.registrations.sign_up'), new_user_registration_path %>
-      <%= menu_item t('devise.registrations.sign_in'), new_user_session_path %>
+      <%= menu_item t('devise.registrations.sign_in'), new_user_session_path, class: 'btn btn-default'%>
```

Comment:
What about a German translation for `devise.regsitrations.sign_in`? Maybe that way, we can take a first step to integrating German translations gradually. I found some sample German translations of the entire devise locales YAML here: https://github.com/plataformatec/devise/wiki/I18n

Reply:
Here is the PR with the locales #111